HW4.5.java

```diff
@@ -1 +1,32 @@
+package com.company;
 
+import java.util.Scanner;
+
+public class SqrtApp {
+
+    public static void main(String[] args) {
+        SqrtApp start = new SqrtApp();
+        Scanner input = new Scanner(System.in);
+        double number = input.nextDouble();
+        double result = start.sqrt(number);
+        System.out.println(result);
+    }
+
+    public static double sqrt(double x) {
```

Comment:
simplify
you don't need realize your own sqrt
you can use Math.sqrt()

Reply:
Ok